protocols/minigames/chickenshock/2022-08-08-protocol.md

```diff
@@ -0,0 +1,30 @@
+# Meeting Transcript 08.08.2022
+
+What do we still want to add to chickenshock / what would be nice to have, what is left to do this sprint?
+
+- present: Martin, Levi
+- absent excused: -
+- absent unexcused: -
+
+## What do we want to add?
+
+- Pause menu with options menu and quit button.
+- If the player pauses he loses 1 point per question to prevent abuse of pausing
```

Comment:
Alternatively, the question could be skipped.
Or the lecturer can set delays per configuration to show a question only for the set amount of time.

Reply:
Not a bad idea, we will keep it in mind